resources/js/app.js

```diff
@@ -3,9 +3,14 @@
  * includes Vue and other libraries. It is a great starting point when
  * building robust, powerful web applications using Vue and Laravel.
  */
+// main.js
+import $ from 'jquery'
+import 'slick-carousel'
 
-require('./bootstrap');
+window.jQuery = window.$ = $;
```

Comment:
dòng này để làm gì thế bro, tưởng jquery là dùng luôn chứ nhỉ ?

Reply:
cái này tớ tìm hiểu trên StackOverFlow và cóp nguyên nên cũng chưa rõ bạn có thể lên đây tìm hiểu ạ

https://stackoverflow.com/questions/60366743/how-to-install-slick-carousel-using-npm-and-gulp